frontend/src/component/common/Markdown/Markdown.tsx

```diff
@@ -19,6 +19,9 @@ const LinkRenderer = ({
     );
 };
 
-export const Markdown = (props: ComponentProps<typeof ReactMarkdown>) => (
-    <ReactMarkdown components={{ a: LinkRenderer }} {...props} />
+export const Markdown = ({
+    components,
+    ...props
+}: ComponentProps<typeof ReactMarkdown>) => (
+    <ReactMarkdown components={{ a: LinkRenderer, ...components }} {...props} />
```

Comment:
For anyone coming across this: This was part of an experiment where we customized `em` components within our markdown renderer, in this case specifically for the timeline event tooltip. 

Even though we ended up choosing a different approach, we still see this as an improvement (scouting) that could be helpful for future cases where we need to override specific components in the same way.